shared/common-adapters/icon.constants.js

```diff
@@ -13,197 +13,190 @@ const iconMeta_ = {
   'iconfont-caret-down': {
     isFont: true,
     gridSize: 8,
-    charCode: 59648,
```

Comment:
I fixed the regex on our end that was broken by icomoon's JSON change, so now we have charcodes back.